### PR TITLE
Refactor p-j using pj.c

### PR DESCRIPTION
**Pull request by a-meth**

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -2719,13 +2719,18 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 		return 0;
 	}
 
+	PJ *pj = pj_new ();
+	if (!pj) {
+		return 1;
```
Comment (radare) [inline libr/core/cmd_print.c]: I think it should return 0. As an error code

File: libr/core/cmd_print.c [hunk 2]
```diff
@@ -2741,26 +2746,24 @@ static int cmd_print_blocks(RCore *core, const char *input) {
...
 				|| (as->block[p].comments)
 				|| (as->block[p].symbols)
 				|| (as->block[p].perm)
 				|| (as->block[p].strings)) {
-				r_cons_printf ("\"offset\":%"PFMT64u ",", at);
-				r_cons_printf ("\"size\":%"PFMT64u ",", piece);
+				pj_kn (pj, "offset", at);
+				pj_kn (pj, "size", piece);
 			}
 #define PRINT_VALUE(name, cond, v) { \
```
[the remark this answers is not in the document]
Comment (a-meth) [reply libr/core/cmd_print.c]: So there would be multiple condition checks?

**Comment by radare** [comment]
Yes. But it doesnt hurts if we remove the ifs imho

**Comment by a-meth** [comment]
You mean to print them directly without nullchk?

**Pull request merged**